onnxruntime/test/providers/qnn/layer_norm_test.cc

```diff
@@ -15,124 +16,154 @@
 namespace test {
 #if defined(__aarch64__) || defined(_M_ARM64) || defined(__linux__)
 
-static void RunLayerNormCpuTest(const std::vector<int64_t>& shape) {
+// Returns a function that creates a graph with a single LayerNormalization operator.
+static GetTestModelFn BuildLayerNormTestCase(const TestInputDef<float>& input_def,
```

Comment:
>BuildLayerNormTestCase [](start = 22, length = 22)

Is it possible to re-use RunQDQOpTest with const std::vector<TestInputDef<float>>& input_defs, which you already used for simple op test?